visual-js/visual-wdio/src/SauceVisualService.ts

```diff
@@ -406,6 +409,8 @@ export default class SauceVisualService implements Services.ServiceInstance {
       const jobId = (browser.capabilities as any)['jobUuid'] || sessionId;
       const result = await api.createSnapshotFromWebDriver({
         captureDom: options.captureDom ?? this.captureDom,
+        clipElement:
+          options.clipElement?.elementId ?? this.clipElement?.elementId,
         clipSelector: options.clipSelector ?? this.clipSelector,
```

Comment:
Should we at the same time make the `clipSelector` use `clipElement` behind the scene?

@omacranger what do you think ?

Reply:
Yeah I think that would be fine to do now. They would both throw the same exception if it couldn't be found / resolved on the backend so it wouldn't be a breaking behavior.